gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml

```
@@ -101,10 +101,11 @@ http:
 management:
   type: mongodb                  # repository type
   mongodb:                       # mongodb repository
+     uri: mongodb://localhost:27017/gravitee?connectTimeoutMS=2000
```

Shouldn't we use the `db` values ?
```suggestion
     uri: mongodb://${ds.mongodb.host}:${ds.mongodb.port}/${ds.mongodb.dbname}?connectTimeoutMS=2000
```

---

it shouldn’t be committed at all